src/components/__tests__/FormCardMerch.cy.js

```diff
@@ -0,0 +1,125 @@
+import { colors } from 'quasar';
+import FormCardMerch from 'components/form/FormCardMerch.vue';
+import { i18n } from '../../boot/i18n';
+import { rideToWorkByBikeConfig } from 'src/boot/global_vars';
+
+const { getPaletteColor } = colors;
+const black = getPaletteColor('black');
+
+const option = {
```

Comment:
You could move this test data into standalone JSON fixture file, please.

https://medium.com/software-testing-pipeline/cypress-test-fixture-4377aae1d62
https://docs.cypress.io/api/commands/fixture

Reply:
Updated. Thanks for the reference.